src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1709,6 +1709,30 @@ void GraphOptimizer::FuseConvolutionSumAndConvolutionSumActivation(Graph &graph)
         if (mergedConv->isConstant() && !sum->isConstant())
             continue;
 
+        auto checkInputCompatibility = [](NodePtr sumNode) {
+            auto shape1 = sumNode->getInputShapeAtPort(0);
+            auto shape2 = sumNode->getInputShapeAtPort(1);
+            if (!shape1.isCompatible(shape2.getDims())) {
```

Comment:
Isn't it enough to just call `isCompatible` without any additional check? I think it should return false for case: shape1:{128-256} and shape2: {1}.

Reply:
No.
If we only call `isCompatible`, some func tests will fail.
Their shapes are: {-1, 32, -1, -1} and {-1, -1, -1, -1}, the both shapes are incompatible, but we expect they are compatible.

Reply:
Oh, I see. Thanks.
Then lets avoid usage of `isCompatible` at all. The condition inside seems to be good enough.
Also in case shape1 and shape2 ranks are not matched we also should disable fusing.

Reply:
Done.

Reply:
Sorry, updated again.